scripts/template_upload_gce.py

```diff
@@ -67,33 +73,54 @@ def make_kwargs(args, **kwargs):
     return kwargs
 
 
-def download_image_file(image_url, destination=None):
+def make_ssh_client(ssh_host, ssh_user, ssh_pass):
+    connect_kwargs = {
+        'username': ssh_user,
+        'password': ssh_pass,
+        'hostname': ssh_host
+    }
+    return SSHClient(**connect_kwargs)
+
+
+def log_detail(message, provider=None):
+    if provider:
+        print("{} GCE: {}: {}".format(datetime.utcnow(), provider, message))
+    else:
+        print("{} INFO: {}".format(datetime.utcnow(), message))
+
+
+def download_image_file(image_url, ssh_client):
+    """
+    Download the file to the cli-tool-client and return the file path + file name
+    Default destinations are None and are set to the cli-tool-client if left that way
+    :param image_url: string url to the tar.gz image
+    :return: tuple: file_name, file_path strings
+    """
+    # TODO: add a --local option to the script and run commands locally to download
+    # Download to the amazon/gce upload machine in the lab because its fast
+    target_dir = '/root/downloads'
```

Comment:
Possibly, but /tmp only has 1G on this server so that won't be enough.  /var and /root are in the same logical volume, so I could move this to /var.